drivers/clock_control/clock_control_rcar_cpg_mssr.c

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2020 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_rcar_cpg_mssr
+#include <errno.h>
+#include <soc.h>
+#include <drivers/clock_control.h>
+#include <drivers/clock_control/rcar_clock_control.h>
+#include <dt-bindings/clock/renesas_rcar_cpg.h>
+
+struct rcar_mssr_config {
+	uint32_t base_address;
+};
+
+#define DEV_CFG(dev)  ((struct rcar_mssr_config *)(dev->config))
+
+static const uint16_t rmstpsr[] = {
+	0x110, 0x114, 0x118, 0x11c,
+	0x120, 0x124, 0x128, 0x12c,
+	0x980, 0x984, 0x988, 0x98c,
+};
+
+#define RMSTPSR(i)      rmstpsr[i]
```

Comment:
This define is unused.

Reply:
thanks, will use it in next version instead of accessing the array directly.